### PR TITLE
hSphere: Increase max processes on slaves

### PR DESCRIPTION
**Pull request by kratos7**
*\* Scaling clients to large values require increase in
   default kernel values
*\* Make appropriate changes

Signed-off-by: Abdullah Shah abdullah@plumgrid.com

### CODE REVIEW

File: src/main/python/hydro_sphere/vm_files/add_grouping.sh [hunk 1]
```diff
@@ -22,6 +22,15 @@ echo "60mins" | sudo tee /etc/mesos-slave/gc_delay
 # Optionally you can change the mesos work directory from /tmp
 echo "/opt/mesos" | sudo tee /etc/mesos-slave/work_dir
 
+# Increase max number of memory map areas for processes
+# Full doc
+# https://www.kernel.org/doc/Documentation/sysctl/vm.txt
+echo "600000" | sudo tee /proc/sys/vm/max_map_count
```
Comment (sushilks) [inline src/main/python/hydro_sphere/vm_files/add_grouping.sh:28]: This is likely not going to be persistent across boot.
Need to make it persistent.

Comment (sushilks) [inline src/main/python/hydro_sphere/vm_files/add_grouping.sh:28]: Any progress on this ?

Comment (tahir24434) [inline src/main/python/hydro_sphere/vm_files/add_grouping.sh:28]: @kratos7 What do we need here to progress? Do we need to write the settings to /etc/sysctl.conf?

Comment (sushilks) [inline src/main/python/hydro_sphere/vm_files/add_grouping.sh:28]: http://www.cyberciti.biz/faq/making-changes-to-proc-filesystem-permanently/

**Pull request closed without merging**